server/models/Notifications.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const notificationsSchema = new mongoose.Schema({
+  notifications: [
+    {
+      unread: {
+        type: Boolean,
+        default: true,
+      },
+      type: {
+        type: String,
+        default: "message",
+      },
+      title: String,
+      body: String,
+      date: {
+        type: Date,
+        default: Math.floor(Date.now() / 1000),
+      },
```

Comment:
Is there a specific reason you're using epoch time here rather than the built in timestamps from Mongoose?

Reply:
Ah sorry no, I'll change that now.